src/containers/Login/form.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import InputText from 'components/Auth/InputText';
+import ButtonSubmit from 'components/ButtonSubmit';
+
+const Form = () => {
+  return (
+    <div className="row">
+      <form>
+        <InputText
+          label="Email"
+          labelFor="email"
+          inputId="email"
+          inputType="email"
+        />
+
+        <InputText
+          label="Password"
+          labelFor="password"
+          inputId="password"
+          inputType="password"
+          forgotPassLink={ true }
```

Comment:
Do we need this props, no?

Reply:
I think so. What do you suggest? 🤔

Reply:
If it's not used, I think we can safely delete it.

Reply:
All of those props are used in the `InputText` component - https://github.com/rafayet-monon/surveyor/blob/development/src/components/Auth/InputText/index.js

Reply:
Ah yes, but how about rename it as `FormGroup` component link what bootstrap is doing? https://getbootstrap.com/docs/4.0/components/forms/ `InputText` is confusing 😕

Reply:
Rather than naming it `FormGroup` and overriding bootstrap's, I named it `FormInputGroup`. It's less confusing I think.

Reply:
it makes sense.